modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SpringCodegen.java

```diff
@@ -134,7 +134,7 @@ public void processOpts() {
                         (sourceFolder + File.separator + configPackage).replace(".", java.io.File.separator), "WebMvcConfiguration.java"));
                 supportingFiles.add(new SupportingFile("swaggerUiConfiguration.mustache",
                         (sourceFolder + File.separator + configPackage).replace(".", java.io.File.separator), "SwaggerUiConfiguration.java"));
-                supportingFiles.add(new SupportingFile("application.properties",
+                supportingFiles.add(new SupportingFile("application.mustache",

```

Comment:
rename x-spring-paginated OK
move application.mustache to library/springboot OK
create application.mustache to library/springboot OK
Update SpringCodeGen OK
Build OK

remains to be done SpringFox plugin parameter